services/blockchain-indexer/shared/indexer/blockchainIndex.js

```diff
@@ -314,7 +302,7 @@ const deleteIndexedBlocks = async job => {
 };
 
 // Initialize queues
-const indexBlocksQueue = Queue(config.endpoints.cache, 'indexBlocksQueue', indexBlock, 30);
+const indexBlocksQueue = Queue(config.endpoints.cache, 'indexBlocksQueue', indexBlock, 1);
```

Comment:
I have my doubts here, it might introduce slowness, lets test and ensure the index performance.